Source/Scene/Implicit3DTileContent.js

```diff
@@ -393,10 +396,52 @@ function deriveChildTile(
     contentJsons.push(combine(contentJson, implicitTileset.contentHeaders[i]));
   }
 
-  var boundingVolume = deriveBoundingVolume(
-    implicitTileset,
-    implicitCoordinates
-  );
+  var boundingVolume;
+  var childCell;
+  if (defined(implicitTileset.boundingVolume.extensions)) {
+    var boundingVolumeS2 =
+      implicitTileset.boundingVolume.extensions["3DTILES_bounding_volume_S2"];
+    if (defined(boundingVolumeS2)) {
```

Comment:
Could we move this code for doing the S2 child volume derivation to its own function? This will make it easier to handle any merge conflicts with the branch I'm working on.